### PR TITLE
fix(freelancer)

### PR DESCRIPTION
**Pull request by xx-TOOL-xx**
Solved the issue of adding a new freelancer to the table

### CODE REVIEW

**Review by jphinning — APPROVED** [review]

File: src/modules/settings-info/settings-info.module.ts [hunk 1]
```diff
@@ -3,9 +3,10 @@ import { SettingsInfoService } from './settings-info.service';
 import { SettingsInfoController } from './settings-info.controller';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { Users } from 'src/modules/entities/users.entity';
+import { Freelancer } from '../freelancer/entities/freelancer.entity';
```
Comment (jphinning) [inline src/modules/settings-info/settings-info.module.ts]: Absolute path should be used

File: src/modules/settings-info/settings-info.service.ts [hunk 2]
```diff
@@ -1,15 +1,18 @@
 import { Injectable, NotFoundException } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
-import { Users } from 'src/modules/entities/users.entity';
+import { UserRole, Users } from 'src/modules/entities/users.entity';
 import { Repository } from 'typeorm';
 import { CreateSettingsInfoDto } from './dto/create-settings-info.dto';
 import { UpdateSettingsInfoDto } from './dto/update-settings-info.dto';
+import { Freelancer } from '../freelancer/entities/freelancer.entity';
```
Comment (jphinning) [inline src/modules/settings-info/settings-info.service.ts]: Absolute path as well

File: src/modules/settings-info/settings-info.module.ts (hunk 1 above)
Comment (xx-TOOL-xx) [reply src/modules/settings-info/settings-info.module.ts]: fixed

File: src/modules/settings-info/settings-info.service.ts (hunk 2 above)
Comment (xx-TOOL-xx) [reply src/modules/settings-info/settings-info.service.ts]: fixed

**Pull request merged**